examples/maps/classification_one_inch_maps_001/README.md

```diff
@@ -0,0 +1,3 @@
+# Tutorial for Identifying 'Railspace' on 1-inch Ordnance Survey Maps
+
+Work in progress!
```

Comment:
@kmcdono2 Could you please add the content today? I am hoping to merge this PR by tomorrow morning. Thanks.

Reply:
@kmcdono2 aaa, we don't have examples/maps anymore, I will move the README to `examples/geospatial/classification_one_inch_maps_001`

Reply:
Done, here: https://github.com/Living-with-machines/MapReader/pull/38/commits/5dad9fbd0c5ea793668cc462fe6788a05836a0e5